src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapFandom.java

```diff
@@ -0,0 +1,189 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsFandomContent;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.ads.FandomAdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.FandomVideoFanTakeover;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VideoFanTakeover;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsFandomObject;
+import org.testng.annotations.Test;
+
+public class TestAdsVuapFandom extends AdsFandomTestTemplate{
+
+    @Test(
+            dataProviderClass = FandomAdsDataProvider.class,
+            dataProvider = "fandomVuapPage",
+            groups = {"AdsVuapFandomDesktop", "AdsVideoClosedAfterPlayingFandom"}
+    )
+    public void adsVideoClosedAfterPlayingFandom(String pageType, String pageName, String slotName, String iframeId) {
```

Comment:
I rather keep it this way because it will be easier to run correct test if we run test from console